introducing-java/src/test/java/com/onto/javacourse/intro/arrayReverse/ArrayReverseTest.java

```diff
@@ -0,0 +1,25 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class ArrayReverseTest {
+    @Test
+    public void arrayReverseTest() {
+        int[] array = {1, 2, 3, 4, 5, 6, 7};
+        int[] expectedArray = {7, 6, 5, 4, 3, 2, 1};
+        int[] reversedArray = ArrayReverse.arrayReverse(array);
+        assertArrayEquals(expectedArray, reversedArray);
+        Exception emptyArrayException = assertThrows(IllegalArgumentException.class, () -> {
```

Comment:
Each of those should be a new test, but I am gonna merge this for now, just keep in mind for next tasks